auth-jwt/config/config.go

```diff
@@ -0,0 +1,18 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/joho/godotenv"
```

Comment:
Why not built-in _lazy_ load to `.env` file?

Reply:
How would you? I'm still learning golang and could help me to know...

Reply:
I mean this one (from [joho/godotenv](https://github.com/joho/godotenv#usage)):

```go
import (
    // ...

    _ "github.com/joho/godotenv/autoload"
)

// ...

func main() {
    _ = godotenv.Load() // find and load .env life (by default)

   // ...
```

Reply:
Nice to know! Thanks!